mediapipe/tasks/ios/common/utils/sources/MPPCommonUtils.mm

```diff
@@ -0,0 +1,137 @@
+// Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "mediapipe/tasks/ios/common/utils/sources/MPPCommonUtils.h"
+
+#import "mediapipe/tasks/ios/common/sources/MPPCommon.h"
+
+#include <string>
+
+#include "absl/status/status.h"  // from @com_google_absl
+#include "absl/strings/cord.h"   // from @com_google_absl
+
+#include "mediapipe/tasks/cc/common.h"
+
+/** Error domain of MediaPipe task library errors. */
+NSString *const MPPTasksErrorDomain = @"org.mediapipe.tasks";
```

Comment:
```suggestion
NSString *const MPPTasksErrorDomain = @"dev.mediapipe.tasks";
```

Reply:
Actually, we discussed internally. We would like to use "com.google.mediapipe.tasks"